### DISCUSSION TITLE
Question 95: error distance for Sites records [ZK - Action]

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#693] [Q&A]
Description: If I wanted to add the error distance to some site records, would that go in the Radius field on the Lat/Long tab? Should I fill in Verbatim (LatRadiusVerbatim_tab) and Numeric (LatRadiusNumeric_tab)? Are these data published to GBIF?

**Comment by rondlg** [comment]
Response:  
Use Numeric if e.g. “1km” 
	[will check if on the ipt to make sure that it's mapped to GBIF]

**Reply by zkachian** [reply]
I just added a bunch of radii to the LatRadiusNumeric_tab field so I wanted to follow up on this to check if this column is mapped to GBIF? No rush on this though. Thanks!
